client/lib/plugins/utils.js

```diff
@@ -240,7 +240,8 @@ export function normalizePluginData( plugin, pluginData ) {
 						item[ '2x' ] ||
 						item[ '1x' ] ||
 						item.svg ||
-						item.default;
+						item.default ||
+						item;
```

Comment:
The new backend returns the icon URL directly instead of wrapping it under the specific key.